templates/al2023/template.json

```diff
@@ -183,7 +182,6 @@
         "KUBERNETES_BUILD_DATE={{user `kubernetes_build_date`}}",
         "KUBERNETES_VERSION={{user `kubernetes_version`}}",
         "PAUSE_CONTAINER_VERSION={{user `pause_container_version`}}",
-        "PULL_CNI_FROM_GITHUB={{user `pull_cni_from_github`}}",
```

Comment:
We need to get rid of `CNI_PLUGIN_VERSION` and the associated template var as well

Reply:
ack, done